samples/MusicStore/Program.cs

```diff
@@ -15,8 +16,10 @@ public static void Main(string[] args)
                 .AddEnvironmentVariables(prefix: "ASPNETCORE_")
                 .Build();
 
+            var currentAssemblyLocation = new FileInfo(typeof(Program).GetTypeInfo().Assembly.Location);
+
             var builder = new WebHostBuilder()
-                .UseContentRoot(Directory.GetCurrentDirectory())
+                .UseContentRoot(currentAssemblyLocation.Directory.FullName)
```

Comment:
> When the NanoServer tests get executed they currently do not run from the same directory as where the published output

Let's change the tests and not break VS. People use this for demos an stuff.

Examples:
```xml
<ItemGroup>
  <None Include="notes.txt" CopyToOutputDirectory="Always" />
  <!-- CopyToOutputDirectory = { Always, PreserveNewest, Never } -->

  <Content Include="files\**\*" CopyToPublishDirectory="PreserveNewest" />
  <None Include="publishnotes.txt" CopyToOutputDirectory="Always"  CopyToPublishDirectory="Always" />
  <!-- CopyToPublishDirectory = { Always, PreserveNewest, Never } -->
</ItemGroup>
```
more details: http://www.natemcmaster.com/blog/2017/01/19/project-json-to-csproj/#files